app/views/content/landing/food-for-thought/_quotes.html.erb

```diff
@@ -12,31 +12,31 @@
           image: "media/images/content/food-for-thought/Ella_Tyrrell.jpg",
           heading: "Find your people",
           text: "Stay true to who you are and don't change to try and fit in. There'll be plenty of people who will like you for who you are already. Explore the new possibilities there are at uni and find your people. It is your place to have freedom and make it what you want it to be.",
-          accreditation: ["Ella Tyrrell"],
+          accreditation: "Ella Tyrrell",
```

Comment:
@gazcarr The quotes on the ITRFM page are in bold and the link text below in non-bold

This means that on the food-for-thought page, we now have bold quotes and a non-bold name, rather than the reverse which is in prod. Do we really want so much bold text in a block on both pages?